nucleus/common/internal-api/src/main/java/fish/payara/internal/notification/admin/BaseSetNotifierConfigurationCommand.java

```diff
@@ -92,6 +92,7 @@ public abstract class BaseSetNotifierConfigurationCommand<C extends PayaraNotifi
     @Param(name = "enabled")
     protected Boolean enabled;
 
+    @Deprecated
     @Param(name = "noisy", optional = true, defaultValue = "true")
```

Comment:
Deprecated is only a note for the IDE - you probably want to add the `obsolete` parameter to the `Param` annotation too